src/main/kotlin/com/infusionvlc/somniumserver/dreams/models/DreamRequest.kt

```diff
@@ -3,7 +3,8 @@ package com.infusionvlc.somniumserver.dreams.models
 data class DreamRequest(
   val title: String = "",
   val description: String = "",
-  val dreamtDate: Long = 0
+  val dreamtDate: Long = 0,
+  val tags: List<String> = emptyList()
```

Comment:
Great! But I would make a typealias for improving this type:

`typealias DreamRequestTag = String` (In lack of a better name). This way we would have a list of `DreamRequestTag`s instead of just primitive Strings. Kotlin 1.3 introduces [inline classes](https://kotlinlang.org/docs/reference/inline-classes.html) for this kind of situations. I have to read more about them but this seems like a good usecase for them.